assets/js/linear_algebra/dot_product_symmetric.js

```diff
@@ -15,62 +15,21 @@ let origin = [150, 140],
 
-let w_unit = 1.0, h_unit = 1.0,
-    dws_array = [0.4, 0.7, 0.7, 0.7],
-    dhs_array = [0.4, 0.4, 0.4, 0.4];
+let w_unit = 1.0, h_unit = 1.0;
 
-let start_coord_x=(375 - origin[0])/scale, 
-    start_coord_y=(75 - origin[1])/scale;
+let start_coord_x=(340 - origin[0])/scale, 
+    start_coord_y=(75 - origin[1])/scale,
+    last_col_coord = start_coord_x + 1.8 * w_unit,
+    last_row_coord = start_coord_y + 0.9 * h_unit;
 
-function texts_to_show(u, v){
-  let uTv = lib.dot_product(u, v);
-  return [
-    [
-        [
-            {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-            {text: 'v', text_opacity: 0.7, key: 'v'},
-        ], [
-            {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-            {text: v.coord.x.toFixed(2), text_opacity: 0.7, key: 'xv'},
-        ], [
-            {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-            {text: v.coord.y.toFixed(2), text_opacity: 0.7, key: 'yv'},
-
-        ], [
-            {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-            {text: v.coord.z.toFixed(2), text_opacity: 0.7, key: 'zv'}
-        ], [
-            {text: 'u\u1d40', text_opacity: 0.7, key: 'u'},
-            {text: u.coord.x.toFixed(2), text_opacity: 0.7, key: 'xu'},
-            {text: u.coord.y.toFixed(2), text_opacity: 0.7, key: 'yu'},
-            {text: u.coord.z.toFixed(2), text_opacity: 0.7, key: 'zu'},
-            {text: uTv.toFixed(3), font_size: 15, text_color: 0,
-             text_opacity: 1, key:'uTv'}
-        ]
-    ], [
-        [
-            {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-            {text: 'u', text_opacity: 0.7, key: 'u'}
-        ], [
-            {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-            {text: u.coord.x.toFixed(2), text_opacity: 0.7, key: 'xu'}
-        ], [
-            {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-            {text: u.coord.y.toFixed(2), text_opacity: 0.7, key: 'yu'}
-        ], [
-            {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-            {text: u.coord.z.toFixed(2), text_opacity: 0.7, key: 'zu'},
-        ], [
-            {text: 'v\u1d40', text_opacity: 0.7, key: 'v'},
-            {text: v.coord.x.toFixed(2), text_opacity: 0.7, key: 'xv'},
-            {text: v.coord.y.toFixed(2), text_opacity: 0.7, key: 'yv'},
-            {text: v.coord.z.toFixed(2), text_opacity: 0.7, key: 'zv'},
-            {text: uTv.toFixed(3), font_size: 15, text_color: 0,
-             text_opacity: 1, key:'uTv'}
-        ]
-        ]
-  ]
-}
+let v_name = {text: 'v =', x: last_col_coord,
+              y: start_coord_y - 0.6 * h_unit, key: 'v'},
+    u_name = {text: 'u =', x: last_col_coord,
+              y: start_coord_y - 0.6 * h_unit, key: 'u'},
+    vT_name = {text: 'v\u1d40 =', x: start_coord_x - 0.6 * w_unit,
+               y: last_row_coord + 0.05 * h_unit, key: 'v' },
+    uT_name = {text: 'u\u1d40 =', x: start_coord_x - 0.6 * w_unit,
+               y: last_row_coord + 0.05 * h_unit, key: 'u' };
```

Comment:
instead of `*_name`, we should use `*_cell`, because they are each a cell in a table.